bigtable/admin.go

```diff
@@ -1457,10 +1457,17 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
+	return ac.RestoreTableTo(ctx, ac.instance, table, cluster, backup)
+}
+
+// RestoreTableTo creates a new table by restoring from this completed backup.
+//
+// diffInstance is an instance in which the new table will be restored to.
+// Instance must be in the same project as the project containing backup.
+func (ac *AdminClient) RestoreTableTo(ctx context.Context, diffInstance, table, cluster, backup string) error {
 	ctx = mergeOutgoingMetadata(ctx, ac.md)
-	prefix := ac.instancePrefix()
+	prefix := "projects/" + ac.project + "/instances/" + diffInstance
```

Comment:
Could this go in a function or method as well?

Reply:
yes, this could definitely be refactored in a bunch of places in this file. Mind if I create a tracking issue to refactor after?

Reply:
Sure, totally up to you.

Reply:
Realized this already existed and refactored.